streams/src/commonMain/kotlin/com/mirego/trikot/streams/reactive/Publishers.kt

```diff
@@ -38,7 +40,17 @@ object Publishers {
      */
     @JsName("never")
     fun <T> never(): Publisher<T> {
-        return publishSubject()
+        return object : Publisher<T> {
```

Comment:
Maybe we could have created a NeverPublisher class instead of an anonymous object here

Reply:
I though about it. Any disadvantage of having an anonymous object ? you think it ends up being heavier then an actual class ?

Reply:
Other than code readability and re-usability, I don't think.